docs/alerts/understanding-alerts/alert-types-alarm-vs-template.md

```diff
@@ -0,0 +1,104 @@
+# 1.2 Alert Types: `alarm` vs `template`
+
+A Netdata alert can be defined in two ways, depending on **how broadly you want the rule to apply**:
+
+- **`alarm`** a rule attached to a **specific chart instance**
+- **`template`** a rule attached to a **context**, applied to **all charts that match that context**
```

Comment:
As recently discussed with @ktsaou , we want to get rid of this difference. `alarm` is a subset of `template` and users should only use the latter for any new alert definitions.

For completeness, the expected approach is that we will make a change to the alert definition file format to become YAML, and then rewrite both `alarm` and `template` definitions into a unified `alert` definition.

This document should defocus and discourage `alarm` and define it as a specific subset of `template`.